osmosis-1/osmosis.zone_assets.json

```diff
@@ -1553,6 +1553,13 @@
       "base_denom": "erc20/0x655ecB57432CC1370f65e5dc2309588b71b473A9",
       "path": "transfer/channel-204/erc20/0x655ecB57432CC1370f65e5dc2309588b71b473A9",
       "osmosis_verified": false
+    },
+    {
+      "chain_name": "sge",
+      "base_denom": "usge",
+      "path": "transfer/channel-5485/usge",
+      "osmosis_verified": true,
```

Comment:
Please default osmosis_verified to false, and please remove osmosis_fronteir, which isn't used anymore.

Reply:
@JeremyParish69 Please review again